lib/bech32/test/Codec/Binary/Bech32Spec.hs

```diff
@@ -338,16 +373,28 @@ instance Arbitrary ValidBech32String where
             , (hrpShrunk, udpOriginal)
             , (hrpOriginal, udpShrunk) ]
 
+instance Arbitrary DataPart where
+    arbitrary = do
+        len <- choose (0, 64)
+        DataPart <$> replicateM len arbitrary
+    shrink (DataPart dp)
+        | null dp   = DataPart <$> []
```

Comment:
This will cause quickcheck to loop endlessly. `shrink` should **never** yield its argument!

Reply:
Oops, well spotted! I've (hopefully) fixed this.